src/quote/Quote.spec.ts

```diff
@@ -91,7 +91,23 @@ describe('Claim', () => {
   )
   const invalidPropertiesQuote = invalidPropertiesQuoteData
   const invalidCostQuote = invalidCostQuoteData
+  it('compress and decompress quote objects', () => {
+    const resultCompressQuote = Quote.compressQuote(validQuoteData)
```

Comment:
Change request:
Same naming change request as in my other comments.

Reply:
Updated